packages/core/addon/components/navi-visualizations/line-chart.js

```diff
@@ -206,7 +210,7 @@ export default Component.extend({
 
     return {
       data: {
-        type: get(this, 'chartType'),
+        type: this.c3ChartType,
```

Comment:
should this be added to the cp's dependencies?

Reply:
oops good call done.